doc/user_guide/docker/docker_usage.md

```diff
@@ -142,10 +142,35 @@ port to the same port then you can connect with http://localhost:49494.
 └─┘┴  ─┴┘┴ ┴ ┴ └─┘   ┴ └─┘└─┘┴└─  └┘└─┘┴   ┴  ┴ └─┘┴└─  ┴  ┴ ┴└─┘└─┘└┴┘└─┘┴└──┴┘ o
 
 The default password is "ai-lab".
-To update the password as user root run
+To update the password login to the Docker container as user root and run
```

Comment:
"as a root user" or "the user root"

Reply:
I will pick the latter, as the word "root" is calculated dynamically and ticket #66 requests to change it in the future.